forc/src/cli/commands/test.rs

```diff
@@ -32,6 +32,9 @@ pub struct Command {
     pub test_print: TestPrintOpts,
     /// When specified, only tests containing the given string will be executed.
     pub filter: Option<String>,
+    #[clap(long)]
+    /// Number of threads to utilize while running the tests.
```

Comment:
```suggestion
    /// Number of threads to utilize while running the tests. By default, this is the number of logical CPUs on your system.
```

Should we add a note here about the default that is used if left unspecified?